Week9/joyeeon/Mission/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,20 @@
+import { FaShoppingCart } from "react-icons/fa";
+import type { RootState } from '../../app/store'
+import { useSelector } from "react-redux";
+
+
+const Navbar = () => {
+    const cart = useSelector((state: RootState) => 
+        state.cart.items.reduce((sum, item) => sum + item.amount, 0)
+    );
+
+
+    return (
+        <nav className="fix top-0 bg-blue-900 text-white flex items-center justify-between w-full h-[50px]">
```

Comment:
현재 Navbar에서 `fix` 클래스가 `fixed` 의 오타로 보입니다! 네비게이션 고정을 의도한 것 같은데, 현재는 네비게이션이 고정되지 않고 있습니다. 이를 수정하면 스크롤 시에도 장바구니 정보가 항상 보이도록 할 수 있어 사용성이 개선될 것 같아요!
```suggestion
        <nav className="fixed top-0 bg-blue-900 text-white flex items-center justify-between w-full h-[50px]">
```